### DISCUSSION TITLE
Vulnerable nanoid dependency in css-loader

### DISCUSSION THREAD

**Discussion by lennartkoopmann** [webpack/webpack#15232] [General]
Hi everyone,

`css-loader` is depending on `postcss`, which is depending on a [vulnerable version](https://security.snyk.io/vuln/SNYK-JS-NANOID-2332193) of `nanoid`. It looks like the `postcss` maintainers have recently upgraded their dependency on `nanoid` but not published a new release yet: [postcss/postcss#3636eaf9b749c64c5dbfbaa56d45bc22b3169ff9](https://github.com/postcss/postcss/commit/3636eaf9b749c64c5dbfbaa56d45bc22b3169ff9)

I'm not a Javascript expert and I'm not sure what the best way forward is here but still wanted to make the webpack maintainers aware. Let me know if I can help in any way.

Thanks,
Lennart

**Comment by alexander-akait** [comment]
Update your deps locally, it will solve the problem
